<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1513</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="slider_hmin">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>179</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_hmax">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>179</number>
   </property>
   <property name="value">
    <number>179</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_smin">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_smax">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_vmin">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slider_vmax">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_hsvImgDisplay">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>1240</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>H min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>H max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>S min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>S max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>180</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>V min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>V max</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_last_img">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>850</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next_img">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>850</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_first_img">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>850</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_prev_img">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>850</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cnt">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>850</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save_masks">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>860</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Masks</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hminVal">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hmaxVal">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>90</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>179</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sminVal">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>120</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_smaxVal">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>150</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>255</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_vminVal">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>180</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_vmaxVal">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>210</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>255</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
